include/mbedtls/ssl.h

```diff
@@ -3783,7 +3784,9 @@ void MBEDTLS_DEPRECATED mbedtls_ssl_conf_sig_hashes(mbedtls_ssl_config *conf,
  * \param sig_algs List of allowed IANA values for TLS 1.3 signature algorithms,
  *                 terminated by \c MBEDTLS_TLS1_3_SIG_NONE. The list must remain
  *                 available throughout the lifetime of the conf object. Supported
- *                 values are available as \c MBEDTLS_TLS1_3_SIG_XXXX
+ *                 values are available as \c MBEDTLS_TLS1_3_SIG_XXXX . Using
+ *                 this for TLS 1.2, items in this parameter should be
+ *                 `(HashAlgorithm << 8) | SignatureAlgorithm`.
```

Comment:
Thanks for adding this. Can you also edit the `\brief` line above to remove "for use in TLS 1.3"?

Reply:
Okay.